### ISSUE TITLE
Y axis groups use same labels when data is in different range

### ISSUE THREAD

**Issue by pfroud** [knowm/XChart#539]
This code:
```java
public static void main(String[] args) {
     XYChart myChart = new XYChart(800, 600);
     myChart.setTitle("Multiple Y axes scale bug");
     myChart.getStyler().setYAxisGroupPosition(0, Styler.YAxisPosition.Left);
     myChart.getStyler().setYAxisGroupPosition(1, Styler.YAxisPosition.Right);
     myChart.setXAxisTitle("x");
     myChart.setYAxisGroupTitle(0, "group zero");
     myChart.setYAxisGroupTitle(1, "group one");

     myChart.addSeries("series on group zero", new double[]{1, 2, 3}, new double[]{4, 5, 6})
             .setYAxisGroup(0);

     myChart.addSeries("series on group one", new double[]{1, 2, 3}, new double[]{-100, -200, -300})
             .setYAxisGroup(1);

     new SwingWrapper<>(myChart).displayChart();
  }
```

Produces this chart:

![XChart output showing Y axis bug](https://user-images.githubusercontent.com/9056906/102007964-2b337480-3ce2-11eb-9e77-d52aa6c87fb8.png)

The range of the Y axis on the left (group zero) should be 4 to 6 but actually is 0 to -300.

The expected chart, according to Excel:

![expected chart using Microsoft Excel](https://user-images.githubusercontent.com/9056906/102007982-42726200-3ce2-11eb-98a3-274aa0a24245.png)

The least interesting Excel file ever: [expected chart using Microsoft Excel.xlsx](https://github.com/knowm/XChart/files/5684393/expected.chart.using.Microsoft.Excel.xlsx)

**Comment by timmolter** [comment]
Strange, it works fine for me on the development branch at least. You could use the latest SNAPSHOT jar or wait until the next release which should be soon. 

<img width="800" alt="Screen Shot 2020-12-13 at 10 43 24 AM" src="https://user-images.githubusercontent.com/738972/102008364-14b40a00-3d30-11eb-923c-3079d5acf66f.png">

**Comment by pfroud** [comment]
You are right, it works in the development branch!

The Y-axis behavior is correct in release 3.6.5 but wrong in release 3.6.6. I believe the bug was introduced in [d0bd425f](https://github.com/knowm/XChart/commit/d0bd425fde67fbc899f73051b0808f80c61d400f) and fixed in [4bece78d](https://github.com/knowm/XChart/commit/4bece78dea84a62a3f3f003a62b51c32440f9757).

**Comment by timmolter** [comment]
OK, thanks. I'll be releasing 3.7.0 by the end of the week hopefully.
